_tsml_research_resources/soton/iridis/iridis_python.md

```diff
@@ -174,6 +182,23 @@ To delete one job it’s:
 
 >scancel 11133013_1
 
+## An example workflow
+>sinteractive
+
+>cd tsml-eval/_tsml_research_resources/soton/iridis/
+
+>select & modify one of the experiment shell scripts to run
+
+>chmod +x EXAMPLE_EXPERIMENT.sh
+
+>write slurm script for requesting jobs
+
+>chmod +x EXAMPLE_SLURM.slurm
+
+>sbatch EXAMPLE_SLURM.slurm
+
+>squeue -u USERNAME –format="%12i %15P %20j %10u %10t %10M %10D %20R" -r
```

Comment:
Not how I would recommend running these, and it should be outlines in the steps above.